setup.py

```diff
@@ -497,9 +497,15 @@ def setup_args():
     else:
         tests_require += ['pytest']
 
-    package_data = {
-        'tango.databaseds': ['*.xmi', '*.sql', '*.sh', 'DataBaseds'],
-    }
+    if WINDOWS:
+        package_data = {
+            'tango.databaseds': ['*.xmi', '*.sql', '*.sh', 'DataBaseds'],
+            'tango': ['*.dll'],
+        }
+    else:
+        package_data = {
+            'tango.databaseds': ['*.xmi', '*.sql', '*.sh', 'DataBaseds'],
+        }
```

Comment:
Not critical, but it could be a little simpler:
```suggestion
    package_data = {
        'tango.databaseds': ['*.xmi', '*.sql', '*.sh', 'DataBaseds'],
    }
    if WINDOWS:
        package_data['tango'] = ['*.dll']
```

Reply:
Hi! 
Thanks for the review! I pushed changes with your suggestion.
I tested packages for python 2.7, 3.6 and 3.7 as described in  #355 : 
- create a new virtual environment
- conda install numpy
- pip install "new PyTango package from appveyor"
- python > import tango > it works!

It would be nice to include such test in appveyor. I'll create issue for that later.

Reply:
Thanks for notes on testing.  Even better would be to run the full PyTango test suite in AppVeyor.

Could you please run the the tests manually in your conda test environment?  In this related PR, this [comment](https://github.com/tango-controls/pytango/pull/358#issuecomment-633972765) noted that there were only errors when actually exercising a device.

Reply:
@wkitka I have stupid question - how does one do: pip install "new PyTango package from appveyor"?

Reply:
Unfortunately event subscription fails with errors:
- Python 2.7:
```
PyTango.DevFailed: DevFailed[
DevError[
    desc = Device  not found
  origin = Util::get_device_by_name()
  reason = API_DeviceNotFound
severity = ERR]

DevError[
    desc = Device  not found
  origin = DServer::event_subscription
  reason = API_DeviceNotFound
severity = ERR]

DevError[
    desc = Failed to execute command_inout on device , command ZmqEventSubscriptionChange
  origin = Connection::command_inout()
  reason = API_CommandFailed
severity = ERR]

DevError[
    desc = Device server send exception while trying to register event
  origin = EventConsumer::connect_event()
  reason = API_DSFailedRegisteringEvent
severity = ERR]
]
Assertion failed: Successful WSASTARTUP not yet performed (..\..\..\src\signaler.cpp:377)
```

- Python 3.7
```
MemoryError 
Assertion failed: Successful WSASTARTUP not yet performed (..\..\..\src\signaler.cpp:377)
```

Looks like it's connected to ZMQ. I'll try to solve that.

Reply:
@andygotz  you need to download artifact from AppVeyor:
https://ci.appveyor.com/project/tiagocoutinho/pytango/builds/33728106
and install it.
Example for Python 3.7:
artifacts: https://ci.appveyor.com/project/tiagocoutinho/pytango/builds/33728106/job/96bh0m43mol99xbi/artifacts
command: pip.exe install pytango-9.3.3.dev0-cp37-cp37m-win_amd64.whl

Reply:
@wkitka I realise that we've never tried the PyTango test suite in Windows before, so maybe it never worked.  You could try the 9.3.1 version for reference.  It would be interesting to know if the problems you see above were there before.

Reply:
I wasn't able to quickly run PyTango test suite on Windows so I used custom script to test event subscription. Event subscription works in PyTango  9.3.1 on Windows. 
There is one interesting thing: I get `Assertion failed: Successful WSASTARTUP not yet performed (..\..\..\src\signaler.cpp:377)` error during closing script (PyTango  9.3.1 on Windows). It probably means that event subscription's errors in this PR aren't related to ZMQ.
I created #369 - PyTango test suite in AppVeyor.

Reply:
@wkitka Thanks.  Does event subscription work with your custom test script and the new build of 9.3.3dev0+ on Windows?

Reply:
Unfortunately, they don't. I'll look into that.